rpms/generate_rpm_package.sh

```diff
@@ -7,7 +7,7 @@
 # and/or modify it under the terms of the GNU General Public
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
-
+set -x
```

Comment:
This line is unnecessary and should be removed.

Reply:
Fixed in d20150a.